academic-services/knowledge-exploration-service/reference-get-histogram.md

```diff
@@ -0,0 +1,223 @@
+---
+title: GET Histogram
+description: Computes attribute value distribution for a structured query expression
+ms.topic: reference
+ms.date: 2020-02-07
+---
+
+# Histogram REST API
+
+The **Histogram** method computes the entities matching a structured query expression and then calculates the distribution of the requested attributes values in the matched entities.
+
+This method is useful for determining the most dominant attribute values in a result set, e.g. finding the most dominant conference Microsoft publishes in (see [examples](#examples) section below).
+
+``` HTTP
+GET http://{serviceName}.{serviceRegion}.cloudapp.azure.com/calchistogram?expr={expr}
+```  
+
+With optional parameters:
+
+``` HTTP
+GET http://{serviceName}.{serviceRegion}.cloudapp.azure.com/calchistogram?expr={expr}&attributes={attributes}&offset={offset}&count={count}
+```  
+
+## URI Parameters
+
+Name | Required | Type | Description
+--- | --- | --- | ---
+`expr` | Required | string | Structured query expression to evaluate for matching entities used in generating histogram. See [structured query expressions](concepts-query-expressions.md) for documentation.
+`attributes` | Optional | string | A list of comma-separated attributes to generate histograms for. See the [entity schema](reference-makes-api-entity-schema.md) for the attributes that can be requested. If no attributes are specified the response will still include the total number of matching entities. <br/><br/>Defaults to an empty string.
+`offset` | Optional | integer | The number of histogram values to skip. <br/><br/>Defaults to 0.
+`count` | Optional | integer | The number of histogram values to return for each attribute. <br/><br/>Defaults to 10.
```

Comment:
>Defaults to 10 [](start = 102, length = 14)

Is there a max value?

Reply:
I'm removing max values for the self-host

---
In reply to: [373230083](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/97#discussion_r373230083) [](ancestors = 373230083)